homeassistant/components/awair/quality_scale.yaml

```diff
@@ -0,0 +1,60 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: todo
+  config-flow: done
```

Comment:
So the `async_step_local_pick` will list all discovered devices in a list, but I believe when you try to manually set up a device nowadays, we already ask you if you want to setup that device or not. Which makes this logic redundant